go.mod

```diff
@@ -1,3 +1,7 @@
 module github.com/Comcast/scte224structs
 
+require (
+    github.com/stretchr/testify v1.6.1
```

Comment:
and here I was enjoying no dependencies, but asserts can be handy I guess

Reply:
yes otherwise, a lot of nil checks (especially because out structs are full of pointer type fields), were making the tests less readable